File: samples/javascript/Unrelated.js
function main() {
  const numbers = [5, 3, 8, 1, 9];
  while (numbers.length > 1) {
    numbers.sort();
    const smallest = numbers.shift();
    numbers[0] = numbers[0] + smallest;
  }

  const data = { total: numbers[0] };
  try {
    console.log(data.missing.value);
  } catch (e) {
    console.log(data.total);
  }
}

main();

File: samples/python/Unrelated.py
import sys


def main():
    numbers = [5, 3, 8, 1, 9]
    while len(numbers) > 1:
        numbers.sort()
        smallest = numbers.pop(0)
        numbers[0] = numbers[0] + smallest

    data = {"total": numbers[0]}
    try:
        print(data["missing"])
    except KeyError:
        print(data["total"])


main()
